Snowflake.Data.Tests/IntegrationTests/SFConnectionIT.cs

```diff
@@ -18,7 +18,7 @@ namespace Snowflake.Data.Tests.IntegrationTests
     using Snowflake.Data.Tests.Mock;
     using System.Runtime.InteropServices;
 
-    [TestFixture]
+    [TestFixture, NonParallelizable]
     class SFConnectionIT : SFBaseTest
```

Comment:
what do you think about applying arrange/act/assert pattern in this file?

Reply:
I think once we deal with those flaky tests we can approach this. For now I didn't want this PR to grow and this class is huge. It should be even split into 2 separate tests. And those who change pooling toggle should be separately treated not to affect biggest bunch of tests